client/gui/gui_boot.cpp

```diff
@@ -527,7 +565,7 @@ class BootWindow : public Fl_Window
  */
 static BootWindow* MakeBootWindow()
 {
-	return new BootWindow(0, 0, 425, 240, "Odamex " DOTVERSIONSTR);
+	return new BootWindow(0, 0, WINDOW_WIDTH, WINDOW_HEIGHT, "Odamex " DOTVERSIONSTR);
```

Comment:
I would like to see the window dimensions eventually be derived dynamically but I'm not sure of the logic to control that. This is at least a step in the right direction.